pom.xml

```diff
@@ -15,6 +15,16 @@
     <doclint>none</doclint>
     <driver.version>4.9.0</driver.version>
     <xml-format.skip>true</xml-format.skip>
+    <dropwizard.version>2.0.12</dropwizard.version>
+    <!--
+      Note that this is currently aligned with the Jetty version that DropWizard transitively depends on.
+      It's probably a good idea to keep the two versions in sync.
+    -->
+    <jetty.version>9.4.30.v20200611</jetty.version>
```

Comment:
There was also some lack of uniformity across modules: some were using DropWizard 2.0.10. I've aligned everything, and the modules that use Jetty directly use the same version as DropWizard.